app/controllers/api/service_dialogs_controller.rb

```diff
@@ -48,6 +49,12 @@ def copy_resource(type, id, data)
       raise BadRequestError, "Failed to copy service dialog - #{err}"
     end
 
+    def orchestration_template_service_dialog_resource(_type, _id, data)
+      validate_orchestration_template_dialog_create_data(data)
+      ot = OrchestrationTemplate.find(data['ot_id'])
```

Comment:
what  if `ot` is nil (not exist in DB)
I think we need to cover it by some error message.

Reply:
`ot` it would be nice to use `orchestration_template` variable.